### ISSUE TITLE
LoraMPI

### ISSUE THREAD

**Issue by scidom** [JuliaStats/Lora.jl#76]
@fredo-dedup I am currently coding a pop-MCMC runner using the power posterior approach of Friel, see also #23. This can run on the basis of the existing code base (and will add it to Lora), yet at the same time I want to run a massive simulation on a cluster with many parallel chains. For this reason I will need to create a standalone job type that works with MPI, but I will submit it as a separate package called LoraMPI (sitting on top of Lora) because I guess we don't want to add the MPI.jl package as a dependency in Lora's REQUIRE file, what is your opinion about this?

**Comment by fredo-dedup** [comment]
Agreed, let's not change Lora to include additional use cases until we can
come up with a better framework (jobs types, architecture, etc.) that allow
us to grow on a more sound foundation.
This experiment should be very interesting though to inform that future
decision !
Le 27 févr. 2015 12:40, "Theodore Papamarkou" <notifications@github.com> a
écrit :

**Comment by scidom** [comment]
Yes, I agree with your views @fredo-dedup. I am currently working on LoramPI because I need it for my own reaserch, and it is an interesting experiment in relation to Lora's development. For example, one thing i realised we currently miss in our `MCLikelihood` is that we have just a single `eval` field for the log-target, which conceptually is likelihood+prior. In some case we need to have both of these fields separately (and on top of them construct their target-sum) - the power posteriors of Friel's population MCMC is one such case. I will file an issue for this.
